lib/uffizzi/cli/connect.rb

```diff
@@ -20,24 +20,26 @@ def list_credentials
     desc 'docker-hub', 'Connect to Docker Hub (hub.docker.com)'
     method_option :skip_raise_existence_error, type: :boolean, default: false,
                                                desc: 'Skip raising an error within check the credential'
+    method_option :update_credentials_if_exist, type: :boolean, default: false
     def docker_hub
       type = Uffizzi.configuration.credential_types[:dockerhub]
-      check_credential_existence(type, 'docker-hub')
+      credentials_exist = credentials_exist?(type)
```

Comment:
there can be just one credential for any credentials' type, so i think `credentials_exist` should be replaced by `credential_exists`
and all other places where you wrote `credentials` should be changed as well